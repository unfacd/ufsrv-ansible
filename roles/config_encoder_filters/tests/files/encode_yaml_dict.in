var1:
  aaa: bbb
  ccc: 123
  ddd: yes
  eee:
    fff: ggg
    hhh:
      iii: jjj
      kkk: yes
      lll:
        - 987.654
        - mmm
