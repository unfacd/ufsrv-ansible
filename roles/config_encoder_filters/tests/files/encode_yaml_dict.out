var1:
  aaa: "bbb"
  ccc: 123
  ddd: true
  eee:
    fff: "ggg"
    hhh:
      iii: "jjj"
      kkk: true
      lll:
        - 987.654
        - "mmm"
